### PR TITLE
Button: using flex for alignments text and icon

### PR DESCRIPTION
**Pull request by EugeniyKiyashko**

### CODE REVIEW

**Review by marker-dao — DISMISSED** [review]

File: testing/renovation-npm/tests/run-and-capture.test.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
 fixture`Run and Capture`
     .page('http://localhost:3000');
 
-test('Sample app renders correctly', async(t) => {
+test.skip('Sample app renders correctly', async(t) => {
```
Comment (ksercs) [inline testing/renovation-npm/tests/run-and-capture.test.js]: Why do we need skip it?

Comment (EugeniyKiyashko) [reply testing/renovation-npm/tests/run-and-capture.test.js]: The are some issues with task. Saving of screenshots is failed. https://teams.microsoft.com/l/message/19:65a642cbf4664d49b989054fa9127e75@thread.skype/1667474437500?tenantId=e4d60396-9352-4ae8-b84c-e69244584fa4&groupId=8c31b3e5-5870-46b1-92a7-180d13b282ed&parentMessageId=1667474437500&teamName=DevExtreme&channelName=DevOps&createdTime=1667474437500&allowXTenantAccess=false

Comment (EugeniyKiyashko) [reply testing/renovation-npm/tests/run-and-capture.test.js]: We unskip this test later.

Comment (EugeniyKiyashko) [reply testing/renovation-npm/tests/run-and-capture.test.js]: Ok, I fixed it https://github.com/DevExpress/DevExtreme/pull/23011/commits/12424f6ce399cb41375cc3d5177d1fd11017f1f9

**Review by Shuvaevmedia — DISMISSED** [review]

**Review by marker-dao — APPROVED** [review]

**Review by Zedwag — APPROVED** [review]

**Review by Shuvaevmedia — APPROVED** [review]

**Review by williamvinogradov — APPROVED** [review]

**Review by Alyar666 — APPROVED** [review]

**Pull request merged**